web-server/src/components/PRTable/PullRequestsTableHead.tsx

```diff
@@ -42,6 +42,61 @@ export const PullRequestsTableHead: FC<PullRequestsTableHeadProps> = ({
   const somePrsSelected =
     selectedPrIds.value.length > 0 && selectedPrIds.value.length < prs.length;
   const allPrsSelected = selectedPrIds.value.length === prs.length;
+  const downloadCsv = () => {
+    if (!prs?.length) return;
+
+    const headers = [
+      'Pull Request',
+      enabledColumnsSet.has('commits') ? 'Commits' : '',
+      enabledColumnsSet.has('lines_changed') ? 'Lines Changed' : '',
+      enabledColumnsSet.has('comments') ? 'Comments' : '',
+      enabledColumnsSet.has('base_branch') ? 'Base Branch' : '',
+      enabledColumnsSet.has('head_branch') ? 'Head Branch' : '',
+      enabledColumnsSet.has('changed_files') ? 'Files Changed' : '',
+      enabledColumnsSet.has('rework_cycles') ? 'Rework Cycles' : '',
+      enabledColumnsSet.has('author') ? 'Author' : '',
+      enabledColumnsSet.has('reviewers') ? 'Reviewer' : '',
+      enabledColumnsSet.has('first_commit_to_open') ? 'Commit to Open' : '',
+      enabledColumnsSet.has('first_response_time') ? 'Response Time' : '',
+      enabledColumnsSet.has('rework_time') ? 'Rework Time' : '',
+      enabledColumnsSet.has('merge_time') ? 'Merge Time' : '',
+      enabledColumnsSet.has('merge_to_deploy') ? 'Merge to Deploy' : '',
+      enabledColumnsSet.has('lead_time') ? 'Lead Time' : '',
+      enabledColumnsSet.has('created_at') ? 'Created At' : '',
+      enabledColumnsSet.has('updated_at') ? 'Updated At' : ''
```

Comment:
The header names and header column names here are effectively duplicated.
Meaning if we ever change the name of a column, we'll have to remember to update it here as well.

I think you have two options:
1. Use the property_name as the column header, which is fine I think. The property names are sufficiently human readable.
2. Enforce type safety on the column header names in both this function and the Table Header.